rotkehlchen/chain/zksync_lite/manager.py

```diff
@@ -598,6 +577,19 @@ def fetch_transactions(
                 f'{address=}, {start_ts=}, {end_ts=} ',
             )
 
+        # Expand queried range to include new start and end timestamps
+        new_queried_range = (start_ts, end_ts) if queried_range is None else (
+            min(queried_range[0], start_ts),
+            max(queried_range[1], end_ts),
+        )
+        ranges = DBQueryRanges(self.database)
+        with self.database.conn.write_ctx() as write_cursor:
+            ranges.update_used_query_range(
```

Comment:
Why update it here? Where do you use this query range? Where is it going to be used?